search-app/src/cmr/search/services/acls/collections_cache.clj

```diff
@@ -4,10 +4,17 @@
             [cmr.common.jobs :refer [defjob]]
             [cmr.common.log :as log :refer (debug info warn error)]
             [cmr.common.cache :as cache]
-            [cmr.common.cache.in-memory-cache :as mem-cache]
             [cmr.common-app.services.search.query-model :as q]
             [cmr.common-app.services.search.query-execution :as qe]
-            [cmr.search.services.acls.acl-results-handler-helper :as acl-rhh]))
+            [cmr.redis-utils.redis-cache :as redis-cache]
+            [cmr.search.services.acls.acl-results-handler-helper :as acl-rhh]
+            [cmr.common.util :as u]
+            [clojure.walk :as walk]
+            [cmr.common.util :as util]))
```

Comment:
You have cmr.common.util twice.  aliased as u and util.  Also these should be listed higher in the list.  Also clojure.walk needs to be listed near the top of the list.

Reply:
I guess if Im going to keep this code I should do just that. Was just a test that turned out to work great.

Reply:
cleaned up, used editor to do sort, all kinds of things looked off